lib/utils.js

```diff
@@ -66,4 +72,21 @@ async function killAppUsingAppName (udid, appName) {
   }
 }
 
-export { detectUdid, getAndCheckXcodeVersion, getAndCheckIosSdkVersion, killAppUsingAppName };
+async function adjustWDAAttachmentsPermissions(perms) {
+  let processedFolders = [];
+  let folders = await fs.glob(WDA_DERIVED_DATA_SEARCH_MASK);
+  for (let folder of folders) {
+    log.debug(`Found WDA derived data folder: ${folder}`);
+    let attachmentsFolder = path.join(folder, WDA_ATTACHMENTS_FOLDER_RELATIVE_PATH);
+    if (await fs.exists(attachmentsFolder)) {
+      log.info(`Setting '${perms}' permissions to ${attachmentsFolder} folder`);
+      await exec('chmod', [perms, attachmentsFolder]);
```

Comment:
I think it would probably be better to use [fs.chmod](https://nodejs.org/api/fs.html#fs_fs_chmod_path_mode_callback) than shelling out